ui/pages/settings/flask/snaps-list-tab/snap-list-tab.container.js

```diff
@@ -0,0 +1,64 @@
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { ENVIRONMENT_TYPE_POPUP } from '../../../../../shared/constants/app';
+import { getEnvironmentType } from '../../../../../app/scripts/lib/util';
+
+import {
+  SNAPS_VIEW_ROUTE,
+  SNAPS_LIST_ROUTE,
+} from '../../../../helpers/constants/routes';
+import { disableSnap, enableSnap } from '../../../../store/actions';
+import SnapListTab from './snap-list-tab.component';
+
+const mapStateToProps = (state, ownProps) => {
+  const { location, history } = ownProps;
+  const { pathname } = location;
+
+  const viewingSnap = Boolean(pathname.match(SNAPS_VIEW_ROUTE));
+  const pathNameTail = pathname.match(/[^/]+$/u)[0];
+  const envIsPopup = getEnvironmentType() === ENVIRONMENT_TYPE_POPUP;
+  const onClick = (_, snap) => {
```

Comment:
Well this is highly unconventional :sweat_smile: 

As @darkwing mentioned, this container/component pattern is deprecated and should not be used going forward. But in that pattern, the `mapStateToProps` function was for mapping state to props, as the title implies. These two functions aren't doing that - they don't use state. They can be embedded directly in the component, there is no reason to construct them here.